src/tests/contract/F2fCriProviderTest.ts

```diff
@@ -0,0 +1,47 @@
+import { Logger } from "@aws-lambda-powertools/logger";
+import { Verifier, VerifierOptions } from "@pact-foundation/pact";
+import path from "path";
+
+const logger = new Logger({
+	logLevel: "DEBUG",
```

Comment:
How come we have debug here but info in UserInfoTest?

Reply:
Updated to INFO.